Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.yml

```diff
@@ -0,0 +1,2407 @@
+    - description: HTTPS service name. Custom or predefined service that defines the port number on which the virtual server receives HTTPS traffic.
+      isArray: false
+      name: https_service
+      required: false
+    - auto: PREDEFINED
+      description: 'Enable HTTP/2.Required when: HTTPSService is not null.'
+      isArray: false
+      name: http2
+      defaultValue: disable
+      predefined:
+      - enable
+      - disable
+      required: false
+    - description: "Enable to allow FortiWeb to use multiple local certificates."
+      isArray: false
+      name: multi_certificate
+      required: false
+    - description: "Required  when:  multi-certificate is enable.Select the multi-certificate file you have created."
+      isArray: false
+      name: certificate_group
+      required: false
+    - description: "Certificate group name. Required  when:  multi-certificate is disable."
+      isArray: false
+      name: certificate
+      required: false
+    - description: "Certificate intermediate group.Required  when: HTTPSService is not null."
+      isArray: false
+      name: intergroup
+      required: false
+    - auto: PREDEFINED
+      description: The name of an intermediate certificate authority (CA) group.
+      isArray: false
+      name: proxy
+      defaultValue: disable
+      predefined:
+      - enable
+      - disable
+      required: false
+    - description: Enable to automatically redirect all HTTP requests to the HTTPS service with the same URL and parameters.
+      name: redirect_to_https
+      required: false
+    - description: Inline web protection profile name.
+      isArray: false
+      name: inline_protection_profile
+      required: false
+    - auto: PREDEFINED
+      description: Enable to override deny and redirect actions defined in the server protection rules for the selected policy. This setting enables FortiWeb to log attacks without performing the deny or redirect action. Disable to allow FortiWeb to perform attack deny/redirect actions as defined by the server protection rules.
+      isArray: false
+      name: monitor_mode
+      defaultValue: disable
+      predefined:
+      - enable
+      - disable
+      required: false
+    - description: "Enable to differentiate uniform resource locators (URLs) according to upper case and lower case letters for features that act upon the URLs in the headers of HTTP requests, such as block list rules, and allow list rules."
+      isArray: false
+      name: url_case_sensitivity
+      required: false
+    - description: Enter a description or other comment. If the comment is more than one word or contains special characters, surround the comment with double quotes ( " ). The maximum length is 999 characters.
+      isArray: false
+      name: comments
+      required: false
+    - auto: PREDEFINED
+      description: "Enable allow FortiWeb to automatically retrieve CA certificates from Let's Encrypt. Supports API version 2 only."
```

Comment:
This description does not make sense based on the predefined and default values.